common/app/filters/RequestLoggingFilter.scala

```diff
@@ -19,6 +19,7 @@ class RequestLoggingFilter extends Filter with Logging with ExecutionContexts {
           response.header.headers.get("X-Accel-Redirect") match {
             case Some(internalRedirect) => s" - internal redirect to $internalRedirect"
             case None => response.header.status match {
+              case 304 => " - 304 Not Modified"
```

Comment:
I realised that any `304` responses were logging "redirect to [location not found]" which isn't that helpful.  There may be other exceptions where the `Location` header doesn't exist, but I think this is the main one.